test/uffizzi/cli/project/compose_test.rb

```diff
@@ -22,6 +25,63 @@ def test_compose_set_success
     assert_requested(stubbed_uffizzi_create_compose)
   end
 
+  def test_compose_set_with_env_vars_success
+    body = json_fixture('files/uffizzi/uffizzi_create_compose_success.json')
+    stubbed_uffizzi_create_compose = stub_uffizzi_create_compose(Uffizzi.configuration.hostname, 201, body, {}, @project_slug)
```

Comment:
Lets refactor this stub. Instead of `stub_uffizzi_create_compose(Uffizzi.configuration.hostname, 201, body, {}, @project_slug)` use the following `stub_uffizzi_create_compose(project_slug, body)`